src/wiki/getting-started/installing-optifine.md

```diff
@@ -12,7 +12,22 @@ Also, if you're using the Fabric mod loader, there's no native support for it, a
 
 Therefore, you should consider the use of OptiFine alternatives whenever possible. [See the wiki page about it](../install-of-alternatives).
 
-## <img src="https://www.optifine.net/favicon.ico" height="20" /> Installing OptiFine
+## <img src="https://www.optifine.net/favicon.ico" height="20" /> Installing OptiFine standalone
+
+This method requires you either have the minecraft version you wish to install OptiFine on top of installed through the vanilla launcher or recreate the folder structure somewhere on your system (/.minecraft/versions/*1.18.2*/*1.18.2*.jar).
+
+Go to <https://optifine.net/downloads> and find your respective Minecraft version (eg. 1.18.2).
+
+Once found, click **mirror**. Now click the **download** button to download your OptiFine jar file.
+
+Execute the OptiFine jar file and change the selected folder location if required, then hit the **"Extract"** button and save the mod file somewhere you'll remember.
+
+Now in the PolyMC launcher edit the instance you wish to install OptiFine on, open the **Version** tab and on the right click **Add to Minecraft.jar**, select the extracted OptiFine mod(the file ending in \_MOD.jar) and confirm.
```

Comment:
```suggestion
Now in PolyMC edit the instance you wish to install OptiFine on, open the **Version** tab and click **Add to Minecraft.jar**, select the extracted OptiFine jarmod (the file ending in \_MOD.jar) and confirm.
```
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/83089242/188480349-c509cd3a-feb9-43a4-a179-2d9442a7ac94.png">

you don't have to right click